IPython/utils/tokenize2.py

```diff
@@ -91,11 +91,7 @@ def maybe(*choices): return group(*choices) + '?'
 Floatnumber = group(Pointfloat, Expfloat)
 Imagnumber = group(r'[0-9]+[jJ]', Floatnumber + r'[jJ]')
 Number = group(Imagnumber, Floatnumber, Intnumber)
-
-if sys.version_info.minor >= 3:
-    StringPrefix = r'(?:[bB][rR]?|[rR][bB]?|[uU])?'
-else:
-    StringPrefix = r'(?:[bB]?[rR]?)?'
+StringPrefix = r'(?:[bB][rR]?|[rR][bB]?|[uU])?'
```

Comment:
if I remember correctly this file is base on Python 3.2 stdlib, does it make sens to check whether the bugs have been fixed upstream on 3.4+ instead of having it diverge more ?

Reply:
I suspect this change is reverting a divergence that was previously introduced. Ultimately it would be good to get rid of it, but that's a more complex job for another day.